pkg/cmd/generate.go

```diff
@@ -28,7 +28,12 @@ import (
 	"k8s.io/client-go/pkg/runtime"
 )
 
-func Generate(files []string) error {
+func Generate(paths []string) error {
+
+	files, err := GetAllYAMLFiles(paths)
```

Comment:
@kadel instead of changing functions here, and getting all YAML files here, I think it makes more sense to do this in `cmd/{apply,create,delete,generate}.go`, similar to the way `if err := ifFilesPassed(InputFiles); err != nil {` is being done in those files.
We can create one `validateFiles` function in `cmd/util.go` which calls `ifFilesPassed` and `GetAllYAMLFiles` since these are common in all of `cmd/{apply,create,delete,generate}.go` files, and just call `validateFiles` in all of them.

It will make this behavior more uniform instead of adding in `func Generate` used by only generate.go and `func ExecuteKubectl` used by {apply,create,delete}.go.

WDYT?